x-pack/plugin/eql/qa/common/src/main/resources/data/eql_date_nanos.data

```diff
@@ -0,0 +1,2080 @@
+[
```

Comment:
No we have mixed ms and ns precision: https://github.com/elastic/elasticsearch/pull/67666/files/13ce3f8e47b56a7cc9b68b465f569b783d7c9f0c#diff-18d38b464889f7ab0e21ca4cffa2a35cc424ae9ced955be20217f0f0ac46c5ceR1956

Instead of manipulating the long timestamp from the original file in code to produce nano precision timestamp, I chose to replicate the file by just changing the timestamp, as I found the first method much more confusing especially when debugging.